tutorials/waypoint-plugin-scaleway/index.mdx

```diff
@@ -73,7 +73,7 @@ A list of all options can be found in [container plugin documentation](https://g
 
 - A Scaleway account logged into the [console](https://console.scaleway.com)
 - Installed [Docker](https://www.docker.com/)
-- Installed [Terraform](https://developer.hashicorp.com/terraform/downloads)
+- Installed [Terraform/OpenTofu](https://developer.hashicorp.com/terraform/downloads)
```

Comment:
Have two different links one for Terraform and the other for OpenTofu

Reply:
```suggestion
- Installed [Terraform](https://developer.hashicorp.com/terraform/downloads) or [OpenTofu](https://opentofu.org/docs/intro/install/)
```